lib/Pipeline.js

```diff
@@ -88,7 +89,7 @@ Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
     }
     addDefaults(gltfWithExtras, options);
     RemoveUnusedProperties.removeAll(gltfWithExtras);
-    generateNormals(gltfWithExtras, options);
+    generateNormals(gltfWithExtras, options);    
```

Comment:
In future PRs, please be mindful of this extra whitespace added to lines like this.